samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+.. _at_client_sample:
+
+nRF9160: AT Client
+##################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem.
+The sample enables you to use an external computer or MCU to send commands to the LTE-M/NB-IoT modem of your nRF9160 DK.
+
+Overview
+********
+
+The AT Client sample acts as a proxy for sending directives to the nRF9160 modem via AT commands.
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+Also see :ref:`modem_info_readme` for information about how an LTE application can interact with the modem.
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
```

Comment:
It should be made clear somewhere that it must be the non-secure version of the board.
Also, the sample works on the Thingy:91 as well, doesn't it?

Reply:
Some information about the non-secure version of the board is already available in most of the samples including AT client as in here: 
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/coap_client/README.html#building-and-running

Is this the information required? 

Also, should we change the requirements section as follows: 

One of the following development boards:
  
- nRF9160 DK board (PCA10090)
- Thingy:91 (PCA20035)

@joakimtoe : Please suggest on specifying Thingy:91 as one of the  development board devices . If we are specifying Thingy:91,  then there are two options: 

- We can continue with the existing "Connecting" and "Testing" sections, by explicitly making it clear that the "Connecting " part and the "Testing" parts are based on nRF9160 DK board as an example. 
-Another option is to include the "Connecting " part and the "Testing" parts for Thingy:91 as well, in which case most of the steps might get repetitive.

Reply:
Yes, something like that would be nice.
Re. the "Connecting" and "Testing" sections, I do not believe there would be any differences between the DK and Thingy.